zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -887,6 +887,11 @@ private void updateNote(NotebookSocket conn, HashSet<String> userAndRoles, Noteb
 
     Note note = notebook.getNote(noteId);
     if (note != null) {
+      if (!notebook.getConf().isZeppelinNotebookCronEnable()) {
+        if (config.get("cron") != null) {
+          config.remove("cron");
```

Comment:
hmm, not sure - is this going to modify the notebook? even if cron is disabled it shouldn't modify the content permanently - if cron is enabled again the note should work as-is as before?

Reply:
The `config` object is coming from WebSocket's `fromMessage`, over here I'm trying to put a check if someone tries to forcefully inject "cron" (or say WebServer is updated but UI is still running on an old version) then remove it (silently).